sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -80,7 +83,7 @@ def get_token_by_key(endpoint: str, hub: str, key: str, **kwargs: Any) -> str:
 
 class WebPubSubServiceClientOperationsMixin(WebPubSubServiceClientOperationsMixinGenerated):
     @distributed_trace
-    def get_client_access_token(self, **kwargs: Any) -> JSON:
+    def get_client_access_token(self, *, webpubsub_client_access: Optional[str] = "Default", **kwargs: Any) -> JSON:
```

Comment:
The name of the parameter and the comment are inconsistent. Why not name the parameter `client_type` using the nomenclature in the comment?

Reply:
So we had a discussion with the C# SDK owner and decided that client_type might be a bit too generic. So we decided to use webpubsub_client_protocol (which I'm going to change now)

Reply:
So internally we can leave it as client_type, but for consistency across the SDKs, we're gonna leave it as `webpubsub_client_protocol `